packages/ozone/src/mod-service/views.ts

```diff
@@ -1,7 +1,13 @@
 import { sql } from 'kysely'
-import { AppBskyActorDefs, AtpAgent } from '@atproto/api'
+import {
+  AppBskyActorDefs,
+  AtpAgent,
+  ComAtprotoRepoGetRecord,
+} from '@atproto/api'
+import { RecordNotFoundError } from '@atproto/api/dist/client/types/com/atproto/repo/getRecord'
```

Comment:
I think you should be able to avoid dipping into `dist/` and instead use `ComAtprotoRepoGetRecord.RecordNotFoundError`.
```suggestion
```